openedx_tagging/core/tagging/rest_api/v1/urls.py

```diff
@@ -0,0 +1,14 @@
+""" Taxonomies API v1 URLs. """
+
+from rest_framework.routers import DefaultRouter
+
+from django.urls.conf import path, include
+
+from . import views
+
+router = DefaultRouter()
+router.register("taxonomies", views.TaxonomyView)
+
+urlpatterns = [
+    path('', include(router.urls))
+]
```

Comment:
Yep, this is correct, but we'll also need a couple more:

* a `openedx_tagging/core/tagging/rest_api/urls.py` file [like this one](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/demographics/rest_api/urls.py) that [includes](https://docs.djangoproject.com/en/4.0/ref/urls/#include) these v1 URLs.
* a `openedx_tagging/core/tagging/urls.py` file that [includes](https://docs.djangoproject.com/en/4.0/ref/urls/#include) the rest_api urls, gives them a root path and sets their [url namespace](https://docs.djangoproject.com/en/4.0/topics/http/urls/#url-namespaces) to `oel_tagging`.